package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dell/clarity-react",
-    "version": "0.23.0",
+    "version": "0.23.1",
```

Comment:
Please update the version by debasing with master and publish new version....

Reply:
ok